src/bisect.cpp

```diff
@@ -89,10 +103,17 @@ std::vector<long> bisect_multilevel(bulk::world& world,
                                     long start,
                                     long end,
                                     interval labels,
-                                    std::mt19937& rng) {
+                                    std::mt19937& rng,
+                                    std::string breaking_mode,
+                                    std::string limit_edge_size,
+                                    std::string simplify_mode) {
 
+    auto time = bulk::util::timer();
     auto H_reduced = pmondriaan::create_new_hypergraph(world, H, start, end);
-
+    //auto w_p = std::vector<long>(2);

```

Comment:
These can be removed I guess

Reply:
We may want to revert to forego simplification at some point after other changes. But I do think we can always go back in the commit history, so you're right.